t/01-pdk/02-log/04-inspect.t

```diff
@@ -3,7 +3,7 @@ use warnings FATAL => 'all';
 use Test::Nginx::Socket::Lua;
 use t::Util;
 
-plan tests => repeat_each() * (blocks() * 4);
+plan tests => repeat_each() * (blocks() * 4 + 6);
```

Comment:
Here, I moved the `+ 6` inside of the `blocks()` multiplication parenthesis. That is because this plan would become incorrect if we were to update `repeat_each()`.